src/script/standard.cpp

```diff
@@ -290,13 +290,24 @@ CScript GetScriptForRawPubKey(const CPubKey& pubKey)
     return CScript() << std::vector<unsigned char>(pubKey.begin(), pubKey.end()) << OP_CHECKSIG;
 }
 
-CScript GetScriptForMultisig(int nRequired, const std::vector<CPubKey>& keys)
+CScript GetScriptForMultisig(int nRequired, const std::vector<CPubKey>& keys, bool fSorted)
 {
-    CScript script;
+    int nEncoded = 0;
+    std::vector<std::vector<unsigned char>> vEncoded;
+    vEncoded.resize(keys.size());
+    BOOST_FOREACH(const CPubKey& key, keys) {
+        vEncoded[nEncoded++] = ToByteVector(key);
+    }
+
+    if (fSorted) {
+        std::sort(vEncoded.begin(), vEncoded.end());
```

Comment:
I *think* this should do what BIP 67 requires, but someone more familiar with C++ and its locale support (or lack thereof) should probably confirm.